marvel-app/coordinator/SearchPageCoordinator.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SearchPageCoordinator.swift
+//  marvel-app
+//
+//  Created by Michal on 11/08/2022.
+//
+
+import Foundation
+import UIKit
+
+class SearchPageCoordinator: Coordinator{
+    var navigationController: UINavigationController?
```

Comment:
Use DI to get all views. Coordinator should pas navigation controller and container. In init use only navigation controller